manifest.webapp

```diff
@@ -33,6 +33,23 @@
       "public": false
     }
   },
+  "services": {
+    "expiration": {
+      "type": "node",
+      "file": "services/expiration/mespapiers.js",
+      "trigger": "@daily between 1pm and 3pm"
+    }
+  },
+  "notifications": {
+    "expiration": {
+      "description": "Notifies the user when a document is about to expire",
+      "collapsible": true,
```

Comment:
What's the impact of setting "collapsible" to true (or on the contrary, to false)? I concede that I have no idea what this option is doing, even after reading the doc.